R/add_rasterly_heatmap.R

```diff
@@ -111,8 +132,8 @@ add_rasterly_heatmap <- function(p,
 
     data %>%
```

Comment:
Well, it is a bit tricky. Since most functions with data argument will call it `data`, e.g. `lm`, `plotly`, `ggplot`, etc. Due to the different namespace, variable `data` will not overwrite function `utils::data()`. But it may cause some confusion when we do the code review.

Reply:
I probably have overly strong opinions on the topic of overloading, but you're correct that the namespacing should avoid collisions here. It's fine to leave it as-is.